nakama/nakama.lua

```diff
@@ -4475,20 +2966,14 @@ end
 --- list_storage_objects2
 -- List publicly readable storage objects in a given collection.
 -- @param client Nakama client.
--- @param collection_str (string) The collection which stores the object..
--- @param user_id_str (string) ID of the user..
--- @param limit_int (number) The number of storage objects to list. Between 1 and 100..
--- @param cursor_str (string) The cursor to page through results from..
+-- @param collection_str (string) The collection which stores the object.
+-- @param user_id_str (string) ID of the user.
+-- @param limit_int (number) The number of storage objects to list. Between 1 and 100.
+-- @param cursor_str (string) The cursor to page through results from.
 -- @param callback Optional callback function.
 -- A coroutine is used and the result returned if no function is provided.
 -- @return The result.
-function M.list_storage_objects2(
-	client
-	,collection_str
-	,user_id_str
-	,limit_int
-	,cursor_str
-	,callback)
+function M.list_storage_objects2(client, collection_str, user_id_str, limit_int, cursor_str,callback)
```

Comment:
If this is different to `list_storage_objects` it needs a clearer name and description. Currently it has the same description and signature. I would prefer to remove this entirely.

Reply:
Both are part of the apigrpc.swagger.json definition. I can add some code to remove one or the other?